marvelscharacters_view.csv

```diff
@@ -0,0 +1,11 @@
+"id","name","alignment","height_cm","weight_kg","place_of_birth"
+1,"Spider-Man","Hero",178,75.75,"Queens"
+2,"Iron Man","Hero",185,102.06,"Sofia"
+3,"Captain America","Hero",188,108.86,"Manhatten"
+4,"Captain Marvel","Hero",180,74.84,"Boston"
+5,"Black Widow","Hero",170,59.42,"Stalingrad"
+6,"Loki","Villain",193,238.14,"Deity"
+7,"Thanos","Villain",201,446.79,"Titan"
+8,"Red Skull","Villain",196,85.73,"Germany"
+9,"Hulk","Hero",244,635.03,"Dayton"
+10,"Thor","Hero",198,290.3,"Norway"
```

Comment:
this file needs to be removed from the repo - the sql file is better because it includes data types, lengths and everything else